x-pack/plugins/apm/scripts/bootstrap_apm/README.md

```diff
@@ -0,0 +1,56 @@
+# Getting started
+
+A script for creating an elastic stack deployment on a remote cluster using `ecctl` and generating APM data using apm-sythtrace tool.
+
+## Prerequisites for running the script
+
+- [ecctl](https://www.elastic.co/downloads/ecctl) installed
+- API key
+  - You can generate a key on [staging](https://staging.found.no/deployment-features/keys)
+- Make sure you have installed Kibana dependencies
+
+## Configuration
+
+Update the following configuration with your own credentials and store it your home directory `$HOME/.ecctl/config.json`
+
+```
+$HOME/.ecctl/config.json
+
+{
+  "host": "https://staging.found.no",
+  "api_key": "API_KEY",
+  "region": "us-west-2",
+  "output": "text",
+  "timeout": 30000000000,
+  "insecure": true
```

Comment:
Out of interest, why skip TLS validation with `"insecure": true`?

Reply:
`ecctl init` generates this as well out of the box, wonder why indeed.